idemp-base/src/org/bandahealth/idempiere/base/model/MBandaSetup.java

```diff
@@ -1359,6 +1361,44 @@ public boolean configureClientUsers() {
 		});
 		return true;
 	}
+	
+	/**
+	 * Create default business partners for new clients
+	 * @return
+	 */
+	public boolean createDefaultBusinessPartners() {
+		List<MBPartner_BH> businessPartners = new Query(this.context, MBPartner_BH.Table_Name,
+				MBPartner_BH.COLUMNNAME_AD_Client_ID + "=?", getTransactionName()).setParameters(MClient_BH.CLIENTID_CONFIG)
+				.list();
+		businessPartners.forEach((businessPartner) -> {
+			MBPartner instance = new MBPartner(context, 0, getTransactionName());
+			MBPartner.copyValues(businessPartner, instance);
+			if (!instance.save()) {
```

Comment:
Can you set the BPGroup ID to be the one created for this client rather than having it point to the Config client?

Reply:
done